src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.ts

```diff
@@ -51,6 +54,7 @@ export class ProgrammingExerciseUpdateComponent implements OnInit {
     titleNamePattern = '^[a-zA-Z0-9-_ ]+'; // must only contain alphanumeric characters, or whitespaces, or '_' or '-'
     exerciseCategories: ExerciseCategory[];
     existingCategories: ExerciseCategory[];
+    profileInfo: ProfileInfo;
```

Comment:
it would be better to store a boolean "inProductionEnvironment"

Reply:
added and use variables in HTML instead of method calls